protos/sources/blockexplorer/blockexplorer.proto

```diff
@@ -79,19 +79,19 @@ message Transaction {
   uint32 index = 2;
   // The hash of the transaction
   string hash = 3;
-  // The submitter of the transaction (Vega public key)
+  // The Vega public key of the transaction's submitter
   string submitter = 4;
   // The type of transaction
   string type = 5;
-  // The results code of the transaction (0 is success)
+  // The results code of the transaction. 0 indicates the transaction was successful
   uint32 code = 6;
-  // The cursor for this transaction (in the page, used for paginated results)
+  // The cursor for this transaction. This is used for paginating results
   string cursor = 7;
   // The actual command of the transaction
   vega.commands.v1.InputData command = 8;
-  // Submitter's signature of transaction
+  // The submitter's signature of transaction
```

Comment:
I don't actually understand this one - do you know what it's actually trying to express?